### ISSUE TITLE
Feature: Enable Proxy list and block list Backup.

### ISSUE THREAD

**Issue by XingShelly** [celzero/rethink-app#1684]
Recently, I decided to try and delete, rethink and reinstall it again, maybe an issue would get solved (t did not). I thought if I just created a backup and restore it, it would just restore everything.

I noticed that the proxy list and the chosen block list I had in the previous installation were not ported to the new one.

As for the proxey list, I have seen in previous issues that the proxy lists are encrypted with an on-device encryption key and they're not easily portable. However, This was still on the same device and yet it did not work. Also, if used with a password, it should not be an issue, right?

On the other hand, the block lists I have downloaded and handpicked were not backed up as well. I wish if there was an option that would back them up altogether along with the chosen lists.

**Comment by ignoramous** [comment]
> noticed that the proxy list and the chosen block list I had in the previous installation were not ported to the new one

Yeah, we need to impl this. Unfortunately, it is slightly complicated to restore proxies (and not just due to encryption). Track at:
- #1280 

> the block lists I have downloaded and handpicked were not backed up as well

Yikes. The blocklist selection *is* backed-up, but may be lost in the restore process. We'll check

**Comment by XingShelly** [comment]
Just to be clear what I mean when I say block lists. I mean the ones that you can access from the "DNS" tab and then you have to download and manually pick which ones you want. Those are the ones I am talking about, not the rules in the firewall tab.
I don't know if the ones in the DNS tab are also supposed to be backed up or not, but they were not for me. 
Whether it's a bug or a feature, I really hope in the future that we would be able to back up the ones in the DNS ten as well.

**Comment by u17194907425** [comment]
@XingShelly do you know if this is still an issue? looks like the #1280 mentioned above was closed quite some time ago. So maybe this one is also resolved as well?

**Comment by XingShelly** [comment]
It's mixed: 
 - As for the VPN proxy list, it is still not backed up and restored. 
- On the other hand, the DNS block list selection is backed up, even though I have to download the list again (I would have preferred if I did n't have to download them, so I would not make any connections without the block lists being on, but it's mostly just fine.)

**Comment by Onxr** [comment]
V056 WireGuard configs are now working fine but blocklist still isn't backed up.

**Comment by Onxr** [comment]
I took a backup and renamed the extension to rbka and imported to this alpha [release](https://github.com/celzero/rethink-app/actions/runs/34313596448) and I was able to get the blocklist BTW I downloaded the blocklist before restoring.
